airbyte-webapp/src/components/connectorBuilder/manifestToBuilderForm.ts

```diff
@@ -0,0 +1,386 @@
+import isEqual from "lodash/isEqual";
+
+import { AirbyteJSONSchema } from "core/jsonSchema/types";
+import {
+  CartesianProductStreamSlicer,
+  ConnectorManifest,
+  DatetimeStreamSlicer,
+  DeclarativeStream,
+  DeclarativeStreamSchemaLoader,
+  DpathExtractor,
+  HttpRequester,
+  HttpRequesterAuthenticator,
+  InlineSchemaLoader,
+  InterpolatedRequestOptionsProvider,
+  ListStreamSlicer,
+  SimpleRetriever,
+  SimpleRetrieverPaginator,
+  SimpleRetrieverStreamSlicer,
+  Spec,
+  SubstreamSlicer,
+} from "core/request/ConnectorManifest";
+
+import {
+  authTypeToKeyToInferredInput,
+  BuilderFormAuthenticator,
+  BuilderFormValues,
+  BuilderPaginator,
+  BuilderStream,
+  BuilderSubstreamSlicer,
+  DEFAULT_BUILDER_FORM_VALUES,
+  DEFAULT_BUILDER_STREAM_VALUES,
+  isInterpolatedConfigKey,
+} from "./types";
+import { formatJson } from "./utils";
+
+export const convertToBuilderFormValues = (
```

Comment:
Decided to move this and its related methods into its own file, because types.ts was starting to get pretty large